modules/images-editing-image-registry-settings-hcp.adoc

```diff
@@ -0,0 +1,118 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/image-configuration-hcp.adoc
+// * post_installation_configuration/preparing-for-users.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="images-editing-image-registry-settings-hcp_{context}"]
+= Editing image registry settings for {hcp-title}
+
+You can change the image registry config with the `rosa edit` command.
+
+[WARNING]
+====
+When the `allowedRegistries` parameter is defined, all registries, are blocked unless explicitly listed. To prevent pod failure, a list of Red Hat registries is automatically whitelisted, as they are required by payload images within your environment. The current list consists of `image-registry.openshift-image-registry.svc:5000,quay.io,registry.redhat.io` and it is also visible when running `rosa describe cluster`.
+====
+
+[NOTE]
+====
+You can change any registry related parameter which will trigger a rollout across all machinepools (all machinepool nodes will be recreated, following pod draining from each node).
+====
+
+.Procedure
+
+* Update or edit the image registry for the cluster by running the following command:
+
++
+[source,terminal]
+----
+$ rosa edit cluster --registry-config-insecure-registries <insecure_registries> \
+   --registry-config-allowed-registries <allowed_registries> \
+   --registry-config-allowed-registries-for-import <registry_name:insecure> \
+   --registry-config-additional-trusted-ca <additional_trusted_ca_file>
+----
++
+.Example output
+[source,terminal]
+----
+rosa edit cluster -c <cluster_name> --registry-config-allowed-registries <allowed_registries> \
+? Changing any registry related parameter will trigger a rollout across all machinepools \
+(all machinepool nodes will be recreated, following pod draining from each node). \
+Do you want to proceed? Yes \ <1>
+I: Updated cluster '<cluster_name>'
```

Comment:
I don't think we need to show the command again in the "Example output" (since we just showed the command above). We typically want the command and the output in separate code blocks.

Also you don't need the `\` line breaks here (since it's not intended to be actually copy/pasted into a terminal). And there's no matching `<1>` callout, so remove it from the block.

```suggestion
? Changing any registry related parameter will trigger a rollout across all machinepools 
(all machinepool nodes will be recreated, following pod draining from each node). 
Do you want to proceed? Yes
I: Updated cluster '<cluster_name>'
```

Reply:
removed command from output and applied other changes, thanks